odpf/proton/metadata/resource/OptimusLog.proto

```diff
@@ -0,0 +1,87 @@
+syntax = "proto3";
+
+package odpf.proton.metadata.resource;
+
+option java_package = "io.odpf.proton.metadata.resource";
+option java_outer_classname = "OptimusLog";
+option go_package = "github.com/odpf/proton/metadata/resource";
+
+import "google/protobuf/timestamp.proto";
+
+message OptimusJobMetadataKey {
+  string urn = 1;
```

Comment:
Can you give an example what this will look like? How are we mapping it with `tenants`?